client/src/services/favorites.servise.ts

```diff
@@ -0,0 +1,18 @@
+import axiosInstance from "./axios.service";
+
+export const getFavorites = () => {
+    return axiosInstance.get(`/favorites/`)
+        .then((response) => response.data)
+        .catch(console.error);
+}
+
+export const updateFavorites = (data: { itemId: string, action: string }) => {
```

Comment:
`export type ActionType = 'ADD' | 'REMOVE';`
Then use it as a type for `action` here and in `FavoriteIcon`

Reply:
done